# Variacao minima na posicao para que o buffer seja atualizado
d_displacement_limit: 0.01

# Tolerancia entre o ponto tido como objetivo e posicao de parada
d_tolerance: 0.2

# Distancia de retorno maxima
d_return: 5.0

# Topico contendo a odometria do robo
odometria: '/tf'
# odometria: '/robot_pose_ekf/odom_combined'
# odometria: '/realsense/odom/sample'

# Child frame id
# child_frame_id: 'base_link'
child_frame_id: 'realsense_pose_frame'
# child_frame_id: 'EspeleoRobo'
